docs/contents/pdf_model.puml

```diff
@@ -228,10 +233,12 @@
         HorizontalBox "+textbox  0..1" --* Cell
         HorizontalBox "+textboxes  0..*" --* Figure
         HorizontalBox "+textbox  0..1" --* Chapter
+        HorizontalBox "+textboxes  0..*" --* Rect
 
         Paragraph "+b_source  1" *-- "+links  *" Link
         Figure "+b_source  1" *-- "+links  *" Link
         Cell "+b_source  1" *-- "+links  *" Link
+        Rect "+b_source  1" *-- "+links  *" Link
```

Comment:
I think `Rect` cannot be a target of a link. It's just a graphical element.
If we do target search for links I think what people want is a paragraph, table or figure.
So if a `Rect` contains text, it is also a paragraph and the alorithm will find this in the search area.